### PR TITLE
Сleared up the message when a comment is hidden 

### LINKED ISSUE #17832: "Comment hidden by author"
**Describe the bug**

The UI description of when a comment is hidden by the author of the post reads as if it was hidden by the author of the comment.

At first, I thought I had clicked on my comment by accident but then I realized that the author of the post just hid the comments that he didn't like. Another user reached out to me and said:

> I'd like to say sorry you had to hide your comment on [http://dev.to](https://t.co/vqKtNmzkUb). I have the impression you've done that because you were predicting the thread would degenerate.

Clearly, this phrasing doesn't confuse only me!

**Expected behavior**

You could use the same phrasing as [on the page of the very comment](https://dev.to/sylwiavargas/comment/1p7g5): "Comment hidden by post author" or make it even more specific: "Comment hidden by the original post author"

**Screenshots**

[An example of the confusing phrasing](https://dev.to/afif/a-plagiarism-case-involving-css-tricks-and-smashing-magazine-your-thoughts-22m8):
Comment hidden by author

[An example of a better phrasing](https://dev.to/sylwiavargas/comment/1p7g5):
Screen Shot 2022-06-06 at 5 53 17 PM

**Additional context**

I'm happy to submit a PR for this fix 💕

### PR DESCRIPTION
**Pull request by scientia-ac-labore**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
- Сleared up the message when a comment is hidden 
- Adds i18n for "_comment_proper.html.erb" file

## Related Tickets & Documents

- Related Issue #14888
- Closes #17832

### Before 
alt text

### After
Screenshot from 2022-07-21 15-34-25

### UI accessibility concerns?

Change is consistent with the interface.
Screenshot from 2022-07-21 15-36-46

## Added/updated tests?

- [ ] Yes
- [X] No, and this is why: There is no change in logic.
- [ ] I need help with writing tests

### CODE REVIEW

File: config/locales/views/comments/en.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ en:
         submit: Delete
         edit: Edit
         cancel: Dismiss
+        text_html_comment: Comment deleted
```
Comment (aitchiss) [inline config/locales/views/comments/en.yml:31]: We tend to only use the `html_` naming when some amount of HTML is going to be passed to the i18n string, which isn't the case for these new strings.

How about we rename this key something like `comment_deleted: Comment deleted`?  Same for the other new key below, perhaps something like `comment_hidden: Comment hidden by post author`?

**Review by aitchiss — COMMENTED** [review]
Thank you for your work on this! I just have a small suggestion re naming - let me know what you think!

**Comment by Ridhwana** [comment]
This looks great, thank you @scientia-ac-labore. I just wanted to follow up on your thoughts about @aitchiss's suggestion [on the renaming the keys](https://github.com/forem/forem/pull/18199#discussion_r929853963)? 😊

**Comment by scientia-ac-labore** [comment]
> This looks great, thank you @scientia-ac-labore. I just wanted to follow up on your thoughts about @aitchiss's suggestion [on the renaming the keys](https://github.com/forem/forem/pull/18199#discussion_r929853963)?
Dear @Ridhwana, there was a good suggestion. In a couple of days, I am going to make suggested changes and push.

**Comment by Ridhwana** [comment]
> Dear @Ridhwana, there was a good suggestion. In a couple of days, I am going to make suggested changes and push.

That sounds great! Thank you so much @scientia-ac-labore ✨

**Review by Ridhwana — APPROVED** [review]
Thanks for these changes @scientia-ac-labore, it's looking great ✨

**Review by fdocr — APPROVED** [review]
LGTM. Thank you for the contribution @scientia-ac-labore

**Pull request merged**